examples/postgresql/assembly/index.ts

```diff
@@ -0,0 +1,34 @@
+import { postgresql } from "@hypermode/functions-as";
+
+export function queryPeople(): string {
+  return postgresql.query("neon", "select * from people", []);
+}
+
+export function getPerson(name: string): string {
+  const query = "select * from people where name = $1";
+  return postgresql.query("neon", query, [name]);
+}
```

Comment:
This example is to get a _person_, but instead we get a `string` - which the user would have to decide how to parse.  Since we're the ones in control over how the API returns data, we should return it already parsed to the format that the user is expecting.  In other words, the user should not have to think too much about JSON to interpret the results.

To achieve that, the user can pass in an object to be populated as a generic type parameter.  This pattern is already being used on the GraphQL example.

That also gives the user an opportunity to see how to use the results in their own application logic.

In other words:

```suggestion
export function getPerson(name: string): Person {
  const query = "select * from people where name = $1";
  const people = postgresql.query<Person>("neon", query, [name]);
  if (people.length) == 0 {
    throw new Error(`Person not found named ${name}.`);
  }
  if (people.length > 1) {
    throw new Error(`More than one person found named ${name}.`);
  }
  
  return people[0];
}
```